frappe/utils/password.py

```diff
@@ -131,9 +131,9 @@ def create_auth_table():
 	frappe.db.create_auth_table()
 
 def encrypt(pwd):
-	if len(pwd) > 127:
-		# encrypting > 127 chars will lead to truncation
-		frappe.throw(_('Password cannot be more than 127 characters long'))
+	if len(pwd) > 20000:
+		# https://github.com/frappe/frappe/issues/11039
+		frappe.throw(_('Password cannot be more than 20000 characters long'))
```

Comment:
lets not have a max password length. instead lets except an sql exception where the hashed password length exceeds the row size, and then ask user to use a shorter password.

Reply:
if we want to have a max password length, we can add a system setting for it for the people who want to limit their system's password length.